### PR TITLE
[New Rule] [BBR] Expired or Revoked Driver Loaded

### PR DESCRIPTION
**Pull request by w0rk3r**
## Summary

Identifies an attempt to load a revoked or expired driver. Adversaries may bring outdated drivers with vulnerabilities to gain code execution in kernel mode or abuse revoked certificates to sign their drivers.

Currently, our untrusted driver loaded [rule](https://github.com/elastic/detection-rules/blob/main/rules/windows/defense_evasion_untrusted_driver_loaded.toml) ignores these values, which can create blind spots. Pushing this one as BBR to get an idea of the noise that revoked or expired certificates can generate and evaluate promoting this rule or removing the condition from the production one.

### CODE REVIEW

File: rules_building_block/privilege_escalation_expired_driver_loaded.toml [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+references = [
+  "https://docs.microsoft.com/en-us/previous-versions/windows/hardware/design/dn653559(v=vs.85)?redirectedfrom=MSDN"
+]
+risk_score = 21
+rule_id = "d12bac54-ab2a-4159-933f-d7bcefa7b61d"
+severity = "low"
+tags = ["Domain: Endpoint", "OS: Windows", "Use Case: Threat Detection", "Tactic: Defense Evasion", "Rule Type: BBR"]
+timestamp_override = "event.ingested"
+type = "eql"
+
```
[the remark this answers is not in the document]
Comment (w0rk3r) [reply rules_building_block/privilege_escalation_expired_driver_loaded.toml:30]: Oh, I thought I did it 😅

File: rules_building_block/privilege_escalation_expired_driver_loaded.toml [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+maturity = "production"
+min_stack_comments = "New fields added: required_fields, related_integrations, setup"
+min_stack_version = "8.3.0"
+updated_date = "2023/06/26"
+bypass_bbr_timing = true
+
+[rule]
+author = ["Elastic"]
+description = """
+Identifies an attempt to load a revoked or expired driver. Adversaries may bring outdated drivers with vulnerabilities
```
Comment (Aegrah) [inline rules_building_block/privilege_escalation_expired_driver_loaded.toml:13]: The description indentation seems a bit off

Comment (w0rk3r) [reply rules_building_block/privilege_escalation_expired_driver_loaded.toml:13]: Indented at 120 🤔

**Comment by Aegrah** [comment]
Good use for the BBR rule type! If you fix the description it LGTM

**Review by Aegrah — APPROVED** [review]

**Review by eric-forte-elastic — APPROVED** [review]
Manual review looks good to me :+1:

**Pull request merged**